op-batcher/batcher/espresso.go

```diff
@@ -246,6 +298,32 @@ const VERIFY_RECEIPT_TIMEOUT = 4 * time.Second
 // retrying a job that failed to verify the receipt.
 const VERIFY_RECEIPT_RETRY_DELAY = 100 * time.Millisecond
 
+// TODO (Keyao) Update the espresso-network-go repo for better error handling.
+// <https://app.asana.com/1/1208976916964769/project/1209392461754458/task/1210405729138484?focus=true>
+// Evaluate the verification job.
```

Comment:
Add a comment similarly to the one suggested inhttps://github.com/EspressoSystems/optimism-espresso-integration/pull/161/files#r2114105091
That is if there are no error we "Handle", if there is an issue on our side we "Skip" if the verification timeouts we "RetrySubmission" otherwise we "RetryVerification".